packages/ListBox/src/components/Popover/Popover.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import PropTypes from "prop-types";
+import PopoverStyled from "./Popover.styles";
+import useListBox from "../../useListBox";
+
+const propTypes = {
+  children: PropTypes.node, // eslint-disable-line
+  zIndex: PropTypes.number,
+};
+
+const defaultProps = {
+  zIndex: 100,
```

Comment:
There's a `z()` helper that is meant to limit our z-indexes to a max of `7`.
https://github.com/acl-services/paprika/blob/master/packages/Stylers/src/helpers.js#L30-L34

Reply:
do we want to limit it? or you mean as a defaultProp?

Reply:
I think the default prop should be `z(7)` (or lower, preferably).